cub/README.md

```diff
@@ -1,219 +0,0 @@
-Furthermore, to provide coalesced accesses to device memory, these primitives are
-configured to access memory using a striped access pattern (where consecutive threads
-simultaneously access consecutive items) and then <em>transpose</em> the keys into
-a [<em>blocked arrangement</em>](index.html#sec4sec3) of elements across threads.
-
-Once specialized, these classes expose opaque `TempStorage` member types.
-The thread block uses these storage types to statically allocate the union of
-shared memory needed by the thread block.  (Alternatively these storage types
-could be aliased to global memory allocations).
-
-<br><hr>
-<h3>Supported Compilers</h3>
-
-CUB is regularly tested using the specified versions of the following
-compilers. Unsupported versions may emit deprecation warnings, which can be
-silenced by defining CCCL_IGNORE_DEPRECATED_COMPILER during compilation.
-
-- NVCC 11.0+
-- GCC 5+
-- Clang 7+
-- MSVC 2019+ (19.20/16.0/14.20)
-
-<br><hr>
-<h3>Releases</h3>
-
-CUB is distributed with the NVIDIA HPC SDK and the CUDA Toolkit in addition
-to GitHub.
-
-See the [changelog](CHANGELOG.md) for details about specific releases.
-
-| CUB Release               | Included In                             |
-| ------------------------- | --------------------------------------- |
-| 2.0.1                     | CUDA Toolkit 12.0                       |
-| 2.0.0                     | TBD                                     |
-| 1.17.2                    | TBD                                     |
-| 1.17.1                    | TBD                                     |
-| 1.17.0                    | TBD                                     |
-| 1.16.0                    | TBD                                     |
-| 1.15.0                    | NVIDIA HPC SDK 22.1 & CUDA Toolkit 11.6 |
-| 1.14.0                    | NVIDIA HPC SDK 21.9                     |
-| 1.13.1                    | CUDA Toolkit 11.5                       |
-| 1.13.0                    | NVIDIA HPC SDK 21.7                     |
-| 1.12.1                    | CUDA Toolkit 11.4                       |
-| 1.12.0                    | NVIDIA HPC SDK 21.3                     |
-| 1.11.0                    | CUDA Toolkit 11.3                       |
-| 1.10.0                    | NVIDIA HPC SDK 20.9 & CUDA Toolkit 11.2 |
-| 1.9.10-1                  | NVIDIA HPC SDK 20.7 & CUDA Toolkit 11.1 |
-| 1.9.10                    | NVIDIA HPC SDK 20.5                     |
-| 1.9.9                     | CUDA Toolkit 11.0                       |
-| 1.9.8-1                   | NVIDIA HPC SDK 20.3                     |
-| 1.9.8                     | CUDA Toolkit 11.0 Early Access          |
-| 1.9.8                     | CUDA 11.0 Early Access                  |
-| 1.8.0                     |                                         |
-| 1.7.5                     | Thrust 1.9.2                            |
-| 1.7.4                     | Thrust 1.9.1-2                          |
```

Comment:
Are we sure this is information we definitely don't want to retain anywhere else?